marketplace/dropbox-assets/src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+      html,
+      body,
+      div {
+        margin: 0;
+        padding: 0;
+        border: 0;
+        font-size: 100%;
+        font: inherit;
+        vertical-align: baseline;
+        font-family: system, sans-serif;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs" data-app-key="g1i5ql4pfnwohig"></script>
```

Comment:
Any specific reason to use the script instead of importing the npm package?

Reply:
Just following the instruction: https://www.dropbox.com/developers/chooser

There is no official package for Chooser. Official SDK doesn't support it (https://www.npmjs.com/package/dropbox). I guess we're better off with Dropbox CDN.

Reply:
Gotcha, makes sense.